src/ui/index.js

```diff
@@ -66,6 +66,7 @@ export default async (kbnServer, server, config) => {
 
   async function getKibanaPayload({ app, request, includeUserProvidedConfig }) {
     const uiSettings = server.uiSettings();
+    const translations = await uiI18n.getTranslationsForRequest(request);
```

Comment:
It may not be necessary, but if `angular-translate` requires the language code for the translations it's providing, perhaps we could update `getTranslationsForRequest` to return the resolved language code as well.